architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,15 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, projects with 30 minutes of inactivity are placed in
+an idled state with resources scaled to zero. Upon receiving network activity,
+project resources are un-idled (scaled back up). In addition to auto-idling,
+projects must hibernate 18 hours in a 72-hour period. During the hibernation,
+all project resources are given a hard quota of zero (they cannot be scaled up).
```

Comment:
I think to be clearer something like this:

> 
Any project exceeding 54 cumulative quota-hours of usage in a rolling 72-hour period must hibernate for the next 18 hours. Quota-hours are calculated as the maximum between percentage of terminating and non-terminating pod resource quota consumed, multiplied by the running time of those pods. For example, 2 compute pods each using half of the available memory quota for 1 hour will be counted as 1 Quota-hour.